client.go

```diff
@@ -589,6 +615,42 @@ func New(cfg Config) (c Client, err error) {
 		).WithOpenTracing(), nil),
 	}
 
+	c.Identity = &Identity{
+		Acp: identity.New(httptransport.NewWithClient(
+			cfg.IssuerURL.Host,
+			c.apiPathPrefix(cfg.VanityDomainType, "/api/identity/%s"),
+			[]string{cfg.IssuerURL.Scheme},
+			client,
+		).WithOpenTracing(), nil),
+	}
+
+	c.IdentitySelf = &IdentitySelf{
+		Acp: identityself.New(httptransport.NewWithClient(
+			cfg.IssuerURL.Host,
+			c.apiPathPrefix(cfg.VanityDomainType, "/%s/%s"),
```

Comment:
I have a doubt regarding the apiPathPrefix format string.  From `acp/internal/identity/handler/rest/self/api_server.go`:
```
var APIRoutingPathPrefixes = kernel.NewCommonSubrouterPathPrefix("/{tid}/{aid}/identity")
```

Reply:
Updated